gs-rest-client/src/main/java/client/SimpleClient.java

```diff
@@ -118,95 +118,132 @@ public static void fixMyBug(ServerRequest serverRequest, TokenizerBuilder bobThe
                 e.printStackTrace();
             }
 
-            //Read JSON response from the server in a BufferedReader
+            //Read the byte response from the server in a BufferedReader
             BufferedReader br = new BufferedReader(new InputStreamReader((conn.getInputStream())));
-
             String output;
-            System.out.println("Output from Server .... \n\n");
+
+            //Convert input stream of bytes into a JSON string
             StringBuilder returnedJsonStringBuilder = new StringBuilder();
             while ((output = br.readLine()) != null) {
                 returnedJsonStringBuilder.append(output).append("\n");
             }
             returnedJsonStringBuilder.setLength(returnedJsonStringBuilder.length() - 1); //remove extra newline
             String returnedJsonString = returnedJsonStringBuilder.toString();
 
-            if (method.equals("index")) {
-                System.out.println(returnedJsonString.toString());
-                conn.disconnect();
+            //Handle the server's response based on the requested method.
+            handleResponse(returnedJsonString, tokenBuilder, method);
+            conn.disconnect();
 
+        } catch (MalformedURLException e) {
+            e.printStackTrace();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
 
-            // CODE COMES BACK HERE!!!!
-            } else {
-                DatabaseEntryListWrapper bugFixes = mapper.readValue(returnedJsonString,
-                        DatabaseEntryListWrapper.class);
+    /*
+    * Handles the response from gs REST server.
+    * @param json: a JSON string returned from the server
+    * @param tokenBuilder: a TokenizerBuilder object that contains data for harmonizing tokenized data
+    * @param method: the method requested from the server
+    * 
+    * @return void: Simply print returned data so that the user can see it.
+    */
+    public static void handleResponse(String json, TokenizerBuilder tokenBuilder, String method) {
```

Comment:
The newly implemented JSON response handler. Behavior of this handler is dictated by the 'method' parameter using switch-case statements. This method is void because it simply prints information to the console for the user to view. The TokenizerBuilder is necessary for implementing harmonization.